substrate/frame/vesting/src/lib.rs

```diff
@@ -678,6 +686,121 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
+/// Ensure the correctness of the state of this pallet.
+///
+/// The following expectations must always apply.
+///
+/// ## Expectations:
+///
+/// `handle_before_schedule_starts`:
+/// * the locked amount of a vesting schedule must be equal to the
```

Comment:
We use to wrap line at 100 chars, this line and others after seem to be returning too early.

Reply:
cargo fmt doesn't fix this?